README.md

```diff
@@ -8,19 +8,22 @@
 
 TPI is a [Terraform](https://terraform.io) plugin built with machine learning in mind. Full lifecycle management of computing resources (including GPUs and respawning spot instances) from several cloud vendors (AWS, Azure, GCP, K8s)... without needing to be a cloud expert.
 
-- **Provision Resources**: create cloud compute (CPU, GPU, RAM) & storage resources without reading pages of documentation
-- **Sync & Execute**: easily sync & run local data & code in the cloud
-- **Low cost**: transparent auto-recovery from interrupted low-cost spot/preemptible instances
+- **Lower cost with spot recovery**: transparent auto-recovery from interrupted low-cost spot/preemptible instances
+- **No cloud vendor lock-in**: switch between clouds with just one line thanks to unified abstraction
 - **No waste**: auto-cleanup unused resources (terminate compute instances upon job completion/failure & remove storage upon download of results)
-- **No lock-in**: switch between several cloud vendors with ease due to concise unified configuration
+- **Seamless developer experience**: easily sync data & run code with one command, making the cloud feel like a laptop
 
-Supported cloud vendors include:
+Supported cloud vendors [include][auth]:
 
 - Amazon Web Services (AWS)
 - Microsoft Azure
 - Google Cloud Platform (GCP)
 - Kubernetes (K8s)
```

Comment:
technically should fit on one line:

```suggestion
Amazon Web Services (AWS) | Microsoft Azure | Google Cloud Platform (GCP) | Kubernetes (K8s)
```

Reply:
Yes, sure, so nobody notices that Microsoft Azure doesn't have an official acronym[^1]

[^1]: Although the command–line tool is `az`